openedx/core/djangoapps/user_api/views.py

```diff
@@ -34,8 +37,11 @@
 from openedx.core.lib.api.permissions import ApiKeyHeaderPermission
 from student.cookies import set_logged_in_cookies
 from student.views import AccountValidationError, create_account_with_params
+from util.db import outer_atomic
 from util.json_request import JsonResponse
 
+POST_AUTH_LOG = logging.getLogger('edx.post_auth')
```

Comment:
The convention is to use the following:

```suggestion
log = logging.getLogger(__name__)
```

Reply:
Done